capellambse_context_diagrams/collectors/class_tree.py

```diff
@@ -0,0 +1,88 @@
+# SPDX-FileCopyrightText: 2022 Copyright DB Netz AG and the capellambse-context-diagrams contributors
+# SPDX-License-Identifier: Apache-2.0
+"""This submodule defines the collector for the Class-Tree diagram."""
+from __future__ import annotations
+
+import collections.abc as cabc
+import typing as t
+
+from capellambse import helpers
+from capellambse.model.crosslayer import information
+
+from .. import _elkjs, context
+from . import generic, makers
+
+LAYOUT_OPTIONS: _elkjs.LayoutOptions = {
+    "algorithm": "layered",
+    "edgeRouting": "ORTHOGONAL",
+    "elk.direction": "DOWN",
+    "partitioning.activate": True,
+    "edgeLabels.sideSelection": "ALWAYS_DOWN",
```

Comment:
As explained in another comment, some of these settings (specifically `algorithm`, `edgeRouting` and `elk.direction`) will always be overwritten with the values in the `ClassTreeDiagram._create_diagram` method. We should remove them from here to avoid confusion.